ph5/core/tests/test_base.py

```diff
@@ -16,6 +19,42 @@ def initialize_ex(nickname, path, editmode=False):
     return ex
 
 
+def kef_to_ph5(ph5path, nickname, kefpath, keflist, ex=None):
```

Comment:
This file `test_base.py` is for functionallity that can/should be used throughout the project.

This function `kef_to_ph5` is specific to that code and those tests, and should be moved there. You can put it at the top module level of test_ph5tostationxml.py for all of those tests to use.